airbyte-integrations/connectors/source-s3/Dockerfile

```diff
@@ -17,5 +17,5 @@ COPY source_s3 ./source_s3
 ENV AIRBYTE_ENTRYPOINT "python /airbyte/integration_code/main.py"
 ENTRYPOINT ["python", "/airbyte/integration_code/main.py"]
 
-LABEL io.airbyte.version=0.1.28
+LABEL io.airbyte.version=0.1.29
```

Comment:
should this be a minor update / is this a breaking change?

Reply:
no, that's actually a patch change - it doesn't change what's allowed for this field, it just makes sure the UI understands correctly.